icons/sandwich.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <path d="M3 12a1 1 0 0 1 1-1h16a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-2Z"/>
+ <path d="M16.5 19h3.6c.497 0 .9-.448.9-1v-2c0-.552-.403-1-.9-1H3.9c-.497 0-.9.448-.9 1v2c0 .552.403 1 .9 1h7.912"/>
+ <path d="M10.772 4.955 3 11h18l-7.772-6.045a2 2 0 0 0-2.456 0Z"/>
+ <path d="M12.97 19.775 7 15h12.5l-3.745 4.494a2 2 0 0 1-2.786.281Z"/>
```

Comment:
I think I'd widen this out by 1px on each side, slide the cheese slice a bit to the left to mitigate the denseness right of it, add an extra .5px of rounding to the bread slices and also align those mismatched paths:
![image](https://user-images.githubusercontent.com/17746067/199511882-78515b35-7e0d-40fd-b6f8-568d233e8e81.png)
![sandwich](https://user-images.githubusercontent.com/17746067/199511312-a2721266-1a66-4d4e-bef0-54cfc573e5ed.png)

```suggestion
  <path d="M10.64 5.4 2 11h20l-8.64-5.6a2.52 2.52 0 0 0-2.73 0v0Z" />
  <path d="m8 15 4.95 4.19a2 2 0 0 0 2.84-.26L19 15" />
  <path d="M2 11v2.5c0 .83.67 1.5 1.5 1.5h17c.83 0 1.5-.67 1.5-1.5V11" />
  <path d="M12.73 19H3.5A1.5 1.5 0 0 1 2 17.5v-1c0-.83.67-1.5 1.5-1.5h17c.83 0 1.5.67 1.5 1.5v1c0 .83-.67 1.5-1.5 1.5h-4.78"/>
```